### ISSUE TITLE
Adjustments from nodes-base/credentials

### ISSUE THREAD

**Issue by ivov** [ivov/eslint-plugin-n8n-nodes-base#4]
1. Do not flag dotted or hyphenated strings:

```
name: 'api.jotform.com',
name: 'api.eu.mailgun.net',
name: 'scram-Sha-256',
``` 

2. Do not add period to PEM key example:

```
description: 'Use the multiline editor. Make sure it is in standard PEM key format:-----BEGIN PRIVATE KEY-----KEY DATA GOES HERE-----END PRIVATE KEY-----.',
```

3. Omit default check for `hidden` param type:

```
{
	displayName: 'Auth URI Query Parameters',
	name: 'authQueryParameters',
	type: 'hidden',
	default: `user_scope=${userScopes.join(' ')}`,
},
```

4. Omit unneeded backticks check for multiline strings:

```
{
	displayName: 'Access Token',
	name: 'accessToken',
	type: 'string',
	default: '',
	description: `The access token must have the following scopes:
	<ul>
		<li>Create/modify webhooks</li>
		<li>View webhooks</li>
		<li>View surveys</li>
		<li>View collectors</li>
		<li>View responses</li>
		<li>View response details</li>
	</ul>`,
},
```
